### PR TITLE
Add omitting options argument section

### PR DESCRIPTION
**Pull request by Ahmed-Hakeem**
### Description

Add omitting options argument to show js behavior towards it. 
Enhance the description of fileName and lineNumber to be more descriptive.

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/error/error/index.md [hunk 1]
```diff
@@ -76,6 +76,25 @@ try {
...
+
+//passing primitive value 
+Err = new Error("New error message", "");
+console.log(Err.cause) //undefined 
+
+//omitting it 
+Err =new Error("New error message");
+console.log(Err.cause) //undefined 
+```
+
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/error/error/index.md:97]:
````suggestion
### Omitting options argument

The `Error()` constructor only tries to read `options.cause` if `options` is an object — this avoids ambiguity with the other non-standard `Error(message, fileName, lineNumber)` signature, which requires the second parameter to be a string. If you omit `options`, pass a primitive value as `options`, or pass an object without the `cause` property, then the created `Error` object will have no `cause` property.

```js
// Omitting options 
const error1 = new Error("Error message");
console.log("cause" in error1); // false

// Passing a primitive value 
const error2 = new Error("Error message", "");
console.log("cause" in error2); // false

// Passing an object without a cause property
const error3 = new Error("Error message", { details: "http error" });
console.log("cause" in error3); // false
```

````

File: files/en-us/web/javascript/reference/global_objects/error/error/index.md [hunk 2]
```diff
@@ -41,10 +41,10 @@ Error(message, fileName, lineNumber)
       - : A property indicating the specific cause of the error.
         When catching and re-throwing an error with a more-specific or useful error message, this property can be used to pass the original error.
 - `fileName` {{optional_inline}} {{non-standard_inline}}
-  - : The value for the `fileName` property on the created `Error` object.
+  - : The path to the file that raised this error.
     Defaults to the name of the file containing the code that called the `Error()` constructor.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/error/error/index.md:45]:
```suggestion
  - : The path to the file that raised this error, reflected in the {{jsxref("Error/fileName", "fileName")}} property. Defaults to the name of the file containing the code that called the `Error()` constructor.
```

File: files/en-us/web/javascript/reference/global_objects/error/error/index.md [hunk 3]
```diff
@@ -41,10 +41,10 @@ Error(message, fileName, lineNumber)
...
 - `lineNumber` {{optional_inline}} {{non-standard_inline}}
-  - : The value for the `lineNumber` property on the created `Error` object.
+  - : The line number in which error was raised inside the file.
     Defaults to the line number containing the `Error()` constructor invocation.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/error/error/index.md:48]:
```suggestion
  - : The line number within the file on which the error was raised, reflected in the {{jsxref("Error/lineNumber", "lineNumber")}} property. Defaults to the line number containing the `Error()` constructor invocation.
```

File: files/en-us/web/javascript/reference/global_objects/error/error/index.md [hunk 4]
```diff
@@ -41,10 +41,10 @@ Error(message, fileName, lineNumber)
       - : A property indicating the specific cause of the error.
         When catching and re-throwing an error with a more-specific or useful error message, this property can be used to pass the original error.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/error/error/index.md:42]:
```suggestion
      - : A value indicating the specific cause of the error, reflected in the {{jsxref("Error/cause", "cause")}} property. When catching and re-throwing an error with a more-specific or useful error message, this property can be used to pass the original error.
```

**Comment by Josh-Cena** [comment]
Can't commit to this branch so I have to make suggestions instead

**Comment by Ahmed-Hakeem** [comment]
> Can't commit to this branch so I have to make suggestions instead

I've already allowed editing, and now there is a [ commit ]( https://github.com/mdn/content/pull/22707/commits/6ef0b9afbc1602a9833313c2a587f2480e52ec0a ) with your updates

**Comment by Ahmed-Hakeem** [comment]
> Can't commit to this branch so I have to make suggestions instead

Thanks much for your addition ❤️

**Review by Josh-Cena — APPROVED** [review]
Thanks

**Pull request merged**